### PR TITLE
Fix for #90

### PR DESCRIPTION
**Pull request by torfjelde**
An attempt at a fix for #90 .

### CODE REVIEW

**Comment by dlfivefifty** [comment]
I’ll merge when the tests pass

**Comment by dlfivefifty** [comment]
Can you bump the patch version please?

File: src/FillArrays.jl [hunk 1]
```diff
@@ -179,6 +179,9 @@ reshape(parent::AbstractFill, dims::Dims)        = Base._reshape(parent, dims)
 reshape(parent::AbstractFill, dims::Tuple{Integer, Vararg{Integer}})        = Base._reshape(parent, dims)    
 Base._reshape(parent::AbstractFill, dims::Dims) = fill_reshape(parent, dims...)
 Base._reshape(parent::AbstractFill, dims::Tuple{Integer,Vararg{Integer}}) = fill_reshape(parent, dims...)
+# Resolves ambiguity error with `_reshape(v::AbstractArray{T, 1}, dims::Tuple{I})` where `I` is the word-size on the system
+Base._reshape(parent::AbstractFill{T, 1, Axes}, dims::Tuple{Int64}) where {T, Axes} = fill_reshape(parent, dims...)
```
Comment (dlfivefifty) [inline src/FillArrays.jl:183]: This is very non-standard having overloads for both `Int64` and `Int32`: either just do `Tuple{Int}` or do `Tuple{Integer}`

Comment (torfjelde) [reply src/FillArrays.jl:183]: Ah, yeah of course I should use `Int`, sorry. `Integer` isn't specific enough, and forgot `Int` was a thing.

**Pull request merged**